Sources/NostrSDK/Events/NonParameterizedReplaceableEvent.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NonParameterizedReplaceableEvent.swift
+//
+//
+//  Created by Terry Yiu on 12/23/23.
+//
+
+import Foundation
+
+public protocol NonParameterizedReplaceableEvent: ReplaceableEvent {}
+public extension NonParameterizedReplaceableEvent {
+    var replaceableEventCoordinates: EventCoordinates? {
+        guard kind.isNonParameterizedReplaceable, let publicKey = PublicKey(hex: pubkey) else {
+            return nil
+        }
+
+        return EventCoordinates(kind: kind, pubkey: publicKey, relayURL: nil)
+    }
+
+    func replaceableEventCoordinates(relayURL: URL) -> EventCoordinates? {
```

Comment:
Just make `relayURL` optional with a default value of nil. Then you don't need the `var` above at all.

Reply:
I wanted to do that but the issue is that you can't assign default values on the protocol level. In this case, it's the `ReplaceableEvent` protocol. So it makes for a cumbersome call of `event.replaceableEventCoordinates(relayURL: nil)`. Which might be fine, actually. We generally want to encourage specifying a relay for the "gossip model" rather than omitting it.